### PR TITLE
Encrypted attribute in io.cozy.files

### PR DESCRIPTION
**Pull request by paultranvan**
This introduces the `encrypted` field in the io.cozy.files, used to determine if a file is client-side encrypted.

Note this `encrypted` attribute will be simply ignored by the stack as long as this [PR](https://github.com/cozy/cozy-stack/pull/3378) isn't merged.

### CODE REVIEW

File: packages/cozy-stack-client/src/FileCollection.js [hunk 1]
```diff
@@ -542,7 +547,7 @@ class FileCollection extends DocumentCollection {
      * (no size limit since we can use the body for that) and after we use the ID.
      */
     let metadataId
-    let path = uri`/files/${fileId}?Name=${sanitizedName}&Type=file&Executable=${executable}`
+    let path = uri`/files/${fileId}?Name=${sanitizedName}&Type=file&Executable=${executable}&Encrypted=${encrypted}`
```
Comment (trollepierre) [inline packages/cozy-stack-client/src/FileCollection.js:550]:
```suggestion
    let path = uri`/files/${fileId}?Name=${sanitizedName}&Type=file&Executable=${executable}&encrypted=${encrypted}`
```
????

File: packages/cozy-stack-client/src/FileCollection.js [hunk 2]
```diff
@@ -521,7 +519,14 @@ class FileCollection extends DocumentCollection {
    */
   async updateFile(
     data,
-    { executable = false, fileId, name = '', metadata, ...options } = {}
+    {
```
Comment (trollepierre) [inline packages/cozy-stack-client/src/FileCollection.js:522]: let's update the above jsdoc 🙏

File: packages/cozy-stack-client/src/FileCollection.js (hunk 1 above)
Comment (trollepierre) [reply packages/cozy-stack-client/src/FileCollection.js:550]: oups, it looks like a behaviour in cozy-client. 🤔 Strange. But i resolve

**Review by trollepierre — APPROVED** [review]

File: packages/cozy-stack-client/src/FileCollection.js (hunk 1 above)
Comment (paultranvan) [reply packages/cozy-stack-client/src/FileCollection.js:550]: !!!! ?
The stack expects the query-string params to be like this: https://github.com/cozy/cozy-stack/blob/60193a284672216f502804862f44665771f35ffa/docs/files.md#query-string-1

File: packages/cozy-stack-client/src/FileCollection.js (hunk 2 above)
Comment (paultranvan) [reply packages/cozy-stack-client/src/FileCollection.js:522]: I updated the `FileAttributes` with the new `encrypted` file, did I miss something?

Comment (trollepierre) [reply packages/cozy-stack-client/src/FileCollection.js:522]: yes. I suggest to add the parameter inside the jsdoc line 515

Comment (paultranvan) [reply packages/cozy-stack-client/src/FileCollection.js:522]: Hm, sorry I don't get it. Line 515, we have the [FileAttributes](https://github.com/cozy/cozy-client/blob/24b23dabe8b78fa39132281aa52d36991f8178aa/packages/cozy-stack-client/src/FileCollection.js#L515), that includes the description of the `encrypted` field: https://github.com/cozy/cozy-client/blob/24b23dabe8b78fa39132281aa52d36991f8178aa/packages/cozy-stack-client/src/FileCollection.js#L40
Hence, it is already documented, right? https://github.com/cozy/cozy-client/blob/4c1c3089b65987d4a4697cfeab8cd1ca6ba9e2b5/docs/api/cozy-stack-client.md#fileattributes--object

Comment (trollepierre) [reply packages/cozy-stack-client/src/FileCollection.js:522]: all right, oops 😅

**Review by trollepierre — APPROVED** [review]

**Pull request merged**